lib/cli/src/opts.rs

```diff
@@ -33,8 +33,11 @@ fn parse_registry_url(registry: &str) -> Result<url::Url, String> {
 
 #[derive(clap::Parser, Debug, Clone, Default)]
 pub struct ApiOpts {
+    /// The (optional) authorization token to pass to the registry
     #[clap(long, env = "WASMER_TOKEN")]
     pub token: Option<String>,
+
+    /// The registry to interact with
```

Comment:
Maybe say this?
`Change the current registry`

Reply:
I think there's the possibility that there's no "current" registry, right? Maybe `Select the registry`? `Select the registry to interact with`?

Reply:
no there is always a current registry. by default it's registry.wasmer.io (afaik i have never seen a situation where there is no registry specified, but I could be wrong).